src/main/java/io/github/mountainrange/mule/gameplay/WorldMap.java

```diff
@@ -100,7 +88,7 @@ public int countTilesWithMule(Player player, MuleType mule) {
 	 */
 	public Set<Tile> tilesWithMule(Player player, MuleType mule) {
```

Comment:
We could replace `Tile` with `T` everywhere in return types (e.g. here and in `cursorTile`) if we wanted.